pages/topics/gateway-events.mdx

```diff
@@ -1865,6 +1983,131 @@ Sent when a [stage instance](/resources/stage-instance) is updated. The inner pa
+
+| Field             | Type      | Description                                                            |
+| ----------------- | --------- | ---------------------------------------------------------------------- |
+| stream_key        | string    | The [stream key](#stream-key) for the stream                           |
+| rtc_server_id ^1^ | snowflake | The ID of the RTC server for the stream, used when connecting to voice |
+| region            | string    | The region of the stream                                               |
+| viewer_ids        | array     | The IDs of the viewers currently watching the stream                   |
+| paused            | boolean   | Whether the stream is paused                                           |
+
+^1^ Only present in [Stream Create](#stream-create) Gateway events.
+
+###### Stream Key
+
+The stream key is a unique identifier for a stream, represented as a colon-delimited list of values. The first value is the type of stream, followed by the location of the stream, and finally the owner of the stream.
+
+###### Stream Key Structure
+
+| Field       | Type      | Description                             |
+| ----------- | --------- | --------------------------------------- |
+| type        | string    | The [type of stream](#stream-type)      |
+| guild_id?   | snowflake | The ID of the guild being streamed to   |
+| channel_id? | snowflake | The ID of the channel being streamed to |
+| owner_id    | snowflake | The ID of the user who owns the stream  |
+
+###### Stream Type
+
+| Value | Description                                                               |
+| ----- | ------------------------------------------------------------------------- |
+| guild | A stream in a guild voice channel                                         |
+| call  | A stream in a DM call                                                     |
+| test  | A stream used for speed tests, not encountered in regular stream contexts |
+
+###### Example Stream Key
+
+```md
+- guild:839502008108580904:850360749460553769:852892297661906993
+- call:1110739331624210483:852892297661906993
+- test:852892297661906993
+```
+
+###### Example Stream
+
+```json
+{
+  "stream_key": "call:1110739331624210483:852892297661906993",
+  "rtc_server_id": "1278201813102755892",
+  "region": "us-east",
+  "viewer_ids": ["193696591125807105"],
+  "paused": false
+}
+```
+
+#### Stream Create
+
+Sent when a stream is created. The inner payload is a [stream](#stream-object) object.
+
+#### Stream Server Update
+
+Sent when a stream's voice server is updated. This is sent when initially connecting to a stream, and when the current stream instance fails over to a new server.
+
+<Alert type="warn">
+
+A `null` endpoint means that the voice server allocated has gone away and is trying to be reallocated. You should attempt to disconnect from the currently connected voice server, and not attempt to reconnect until a new voice server is allocated.
```

Comment:
What is "gone away"

Reply:
unallocated, misbehaving, etc.
doesn't really matter for the client